x-pack/legacy/plugins/epm/public/components/version.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { RequirementVersion } from '../../common/types';
+import euiStyled from '../../../../common/eui_styled_components';
+
+const CodeText = euiStyled.span`
```

Comment:
```suggestion
const CodeText = styled.span`
```

Reply:
argh!  these changes got lost in the merge.